ddtrace/contrib/falcon/middleware.py

```diff
@@ -55,9 +55,6 @@ def process_response(self, req, resp, resource, req_succeeded=None):
 
         status = resp.status.partition(" ")[0]
 
-        # FIXME[matt] falcon does not map errors or unmatched routes
-        # to proper status codes, so we we have to try to infer them
-        # here. See https://github.com/falconry/falcon/issues/606
```

Comment:
It sounds like we can maybe update the integration code for this one? The issue was closed and the status code should be available at this point.